### DISCUSSION TITLE
Automatically post on social media whenever there are changes in the state of dz open-source

### DISCUSSION THREAD

**Discussion by ZibanPirate** [dzcode-io/dzcode.io#372] [Ideas]
#### Context
Currently going to `dzcode.io/Contribute` or opening the mobile app and browsing to `Contribute` tab, you will see all the updates (PRs & Issues for now) about the listed Algerian open-source projects.

#### The problem
The user must go to the website, or must install the app and open it, to see updates of the state of dz open-source.

#### Solution
Whenever there are updates in the state of dz open-source (new/closed issues, open/merged PRs, new discussions, new releases, ...etc), we capture these changes, and post it to social media.

**Comment by ZibanPirate** [comment]
it would also be nice if we allow users to add their social media channels (their facebook pages, instagram & twitter accounts) and have these changes appear there in form of stories/posts ...etc, this way we don't limit the content for only dzcode channels, but instead have anyone who wants to share, share.

i would love to brainstorm together on this @dzcode-io/maintainers & @dzcode-io/members

**Reply by omdxp** [reply]
I agree, this could be game changing, for mobile application we can add notifications whenever this happen, it's something that could be done with GitHub actions.

**Reply by omdxp** [reply]
https://github.com/wix/react-native-notifications

**Comment by abderrahmaneMustapha** [comment]
i can help with this , and i think that this package is gonna help us https://github.com/octokit/webhooks.js/

in addtion to this github action documentation https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action
